web/src/beta/lib/core/Crust/Widgets/useWidgetAlignSystem.ts

```diff
@@ -18,23 +30,30 @@ export default ({ alignSystem }: { alignSystem: WidgetAlignSystem | undefined })
   //       The reason why we use invisible list is prevent initializing cost.
```

Comment:
Can you update this comment for the invisible widget IDs to be more understandable?

Reply:
I kept this comment as-is from previous code...